extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -688,9 +688,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     this.ampAnalyticsConfig_ = extractAmpAnalyticsConfig(this, responseHeaders);
     this.qqid_ = responseHeaders.get(QQID_HEADER);
     this.troubleshootData_.creativeId =
-        responseHeaders.get('google-creative-id');
+        (/** @type {string} **/ (responseHeaders.get('google-creative-id')));
```

Comment:
question: there is no guarantee that this'll be a string, but the code [consuming](https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js#L126) this assumed so. Adding a dev assert might just start throwing errors which aren't expected.

In other words this might have been null in some cases even today, but never caught. Adding a dev assert might start surfacing this. It is a great thing to have IF we want to catch these, but NOT if we want to ignore these. 
Not sure about the use case here.

Reply:
pinged author, seems like dev assert will be fine